vmdb/app/presenters/tree_builder_ops_rbac.rb

```diff
@@ -28,7 +28,7 @@ def x_get_tree_roots(options)
   def x_get_tree_custom_kids(object_hash, options)
     objects =
         case object_hash[:id]
-        when "u"  then User.in_my_region.all
+        when "u"  then User.in_my_region
```

Comment:
In this instance, note that no-one has shed a tear over the following two lines, which are also now returning a relation.